sdk/monitor/azquery/metrics_client_test.go

```diff
@@ -60,7 +60,7 @@ func TestQueryResource_BasicQuerySuccess(t *testing.T) {
 	testSerde(t, &res)
 	testSerde(t, res.Value[0])
 	testSerde(t, res.Value[0].Name)
-	testSerde(t, res.Value[0].Timeseries[0])
+	testSerde(t, res.Value[0].TimeSeries[0])
 	//testSerde(t, res.Response.Value[0].Timeseries[0].Metadatavalues[0])
```

Comment:
Can we uncomment or delete this?

Reply:
fixed in [d88f6f6](https://github.com/Azure/azure-sdk-for-go/pull/19946/commits/d88f6f6d2295606f697673505d2949655385e83b)